### ISSUE TITLE
Profiler (vaitrace) - ERROR:root:This xmodel does not support fine grained profiling

### ISSUE THREAD

**Issue by flyinged** [Xilinx/Vitis-AI#1279]
I am trying the profiler on the resnet50 example running on the VCK190 board.

The board is loaded with the DPU reference design (https://github.com/Xilinx/Vitis-AI/tree/3.0/dpu). Vitis-AI runtime 3.0 is installed on the target.

The profiler works normally when I run `vaitrace ./resnet50 /usr/share/vitis_ai_library/models/resnet50/resnet50.xmodel`

but if I run the command with the `--fine_grained` option, then I get the following error:

`ERROR:root:This xmodel does not support fine grained profiling`

How does one enable the fine-grained profiling on an xmodel?

**Comment by li900309** [comment]
hi @flyinged ，
Unfortunately, this feature has been removed in 3.5 due to xcompiler optimizations, and the results produced by the release version of the xcompiler will no longer contain fine-grained network information and therefore cannot be profiling

**Comment by flyinged** [comment]
hi @li900309 

Thanks for your answer. Is there an alternative to that? How is one supposed to debug a design, in case a network has a performance much lower than expected? Coarse profiling just does not provide enough information for a proper analysis.

**Comment by kkzqp** [comment]
Did you solved it?

**Comment by flyinged** [comment]
No. Our client has decided that they don't want to use Vitis-AI anymore for
their project. Thus I've stopped with the attempts too.


Il giorno mer 4 ott 2023 alle ore 12:43 kkzqp ***@***.***> ha
scritto:

**Comment by shaibal13** [comment]
Can anyone solve this issue?

**Comment by TiantianZhang** [comment]
Is anyone solved this problem?

The following reference is not work on KV260.

https://github.com/Xilinx/Vitis-AI/tree/v3.5/examples/vai_profiler
https://docs.xilinx.com/r/en-US/ug1414-vitis-ai/Vitis-AI-Profiler?tocId=Luv7e02rhpEpdYpzLa0cpg
